.github/workflows/release.yml

```diff
@@ -437,13 +437,53 @@ jobs:
         path: |
           signed
 
+# ================================
+#           Validate
+# ================================
+  validate:
+    name: Validate installers
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+    needs: [ osx-sign, win-sign, linux-sign ]
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+
+      - name: Build & Install Windows
+        if: contains(matrix.os, 'windows')
+        shell: sh
+        run: |
+          exePaths=$(find ./win-sign/signed/*.exe) &&
+          for path in $exePaths; do
+            start "" "$path" /WAIT //SILENT //NORESTART &&
+            git-credential-manager-core --version
+          done
+
+      - name: Build & Install Linux
+        if: contains(matrix.os, 'ubuntu')
+        run: |
+          debpath=$(find ./linux-sign/*.deb)
+          sudo apt install $debpath
+          git-credential-manager-core configure
+          git-credential-manager-core --version
```

Comment:
Since we know that GCM is not installed _a priori_ on the build machines, this demonstrates that the executable was installed on the `PATH`. Is there any way we could validate the output of `git-credential-manager-core --version` to be sure we have the right version number?

I see this as a really important validation for the Git installer validation, but might be helpful here.

Reply:
If we could print the output that would be great indeed! I think the script task here should do this already, no?

Reply:
I was thinking something like

```
git-credential-manager-core --version >actual
cat >expect <<-EOF
$GCM_VERSION_NUMBER
EOF
cmp expect actual || return 1
```

so we get an automated build failure if they are different.